cal/v1/network/resources/network.py

```diff
@@ -0,0 +1,213 @@
+from cal.v1.utils import validate_driver
+from validate import validate_resource
+# from cal.v1.resource import Resource
+
+
+class NetworkController():
+    # Define support for GET on a collection
+    @validate_driver
+    def index(self, req, drivers):
+        """List all network
+        List all of netowrks on some special cloud
+        with:
+        :Param  req
+        :Type   object Request
+        """
+        result = []
+        for driver in drivers:
+            result.append(driver.list_network(req.params))
+        data = {
+                'action': "index",
+                'controller': "network",
```

Comment:
do not hardcoded like this, please move all (action and controller or anything that you had hard-coded) to a new class that contain only constant or enum.